app/models/cafe_data_by_category.rb

```diff
@@ -0,0 +1,5 @@
+class CafeDataByCategory < ApplicationRecord
```

Comment:
A lot of these changes were me making minor tweaks to the naming so that everything was clear.